x-pack/examples/reporting_example/public/application.tsx

```diff
@@ -14,7 +14,7 @@ import { ReportingExampleApp } from './components/app';
 export const renderApp = (
   coreStart: CoreStart,
   deps: Omit<StartDeps & SetupDeps, 'developerExamples'>,
-  { appBasePath, element }: AppMountParameters
+  { appBasePath, element }: AppMountParameters // FIXME: appBasePath is deprecated
```

Comment:
worth opening an issue for this, adding a blocker label to it and a version in which the appBasePath will be removed

Reply:
https://github.com/elastic/kibana/issues/107254